hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/fs/ozone/TestOzoneFSWithObjectStoreCreate.java

```diff
@@ -293,13 +290,15 @@ public void testMPUFailDuetoDirectoryCreationBeforeComplete()
     Arrays.fill(b, (byte)96);
     ozoneOutputStream.write(b);
 
+    // Need to close ozoneOutputStream before creating directory.
+    // S3MultipartUploadCommitPartRequestWithFSO#getOpenKey makes an
+    // OMFileRequest#getParentID call - which will fail due to existing
+    // directory.
+    ozoneOutputStream.close();
+
```

Comment:
we now need to close the `ozoneOutputStream` before creating new directories, I have mentioned the reason in the code comment. This is a behavior change in the test, could you please review this @rakeshadr?